app/forms/reservation_form.rb

```diff
@@ -9,11 +9,28 @@ class ReservationForm < Reform::Form
   property :license, on: :work_version, default: 'none'
   property :title, on: :work_version
 
+  def deserialize!(params)
+    params['license'] = collection.default_license if collection.default_license && !work_version.license
```

Comment:
Do we need the check for `!work_version.license`?  I thought this form would only show up for new work_versions (and none of them have a license)?

Reply:
Not entirely sure. This form may also be instantiated on update in the WorksController, so I erred on the cautious side.

https://github.com/sul-dlss/happy-heron/blob/main/app/controllers/works_controller.rb#L81